Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ApplyPaalmanPingsCorrection.py

```diff
@@ -49,8 +55,16 @@ def PyInit(self):
         self.declareProperty(name='CanShiftFactor', defaultValue=0.0,
                              doc='Amount by which to shift the container data')
 
-        self.declareProperty(MatrixWorkspaceProperty('OutputWorkspace', '', direction=Direction.Output),
+        self.declareProperty(MatrixWorkspaceProperty('OutputWorkspace', '',
+                             direction=Direction.Output),
                              doc='The output corrections workspace.')
+        self.declareProperty(name='RebinCanToSample',
+                             defaultValue='Rebin Can',
+                             validator=StringListValidator([
+                                 'Rebin Can',
+                                 'Do Not Rebin Can']),
+                             doc=('Enable or disable RebinToWorkspace on ' +
+                                  'CanWorkspace.'))
```

Comment:
Named arguments are not forced by Python so I'd expect there to be users who omit them. Anyway, I don't feel too strongly about this, so I'll make change the property to boolean.